test/shopify.test.js

```diff
@@ -123,7 +133,7 @@ describe('Shopify', () => {
     const url = { pathname: '/test', ...shopify.baseUrl };
     const scope = common.scope;
 
-    afterEach(() => expect(nock.isDone()).to.be.true);
+    afterEach(() => expect(nock.pendingMocks()).to.deep.equal([]));
```

Comment:
It might indeed be useful but the same pattern is used everywhere so for consistency I would keep it as is for now and possibly change it later everywhere in another PR.